setup.cfg

```diff
@@ -5,6 +5,7 @@ description = Cython setuptools integration
 long_description = file: README.md
 long_description_content_type = text/markdown
 keywords = cython setuptools
+python_requires = '~=3.7'
```

Comment:
Just a question for me: What is meaning of `~=` here ? (Non blocking)

Reply:
It means support all python distribution `>= 3.7` and `< 4.x`